Framework/DataHandling/inc/MantidDataHandling/LoadInstrument.h

```diff
@@ -75,10 +69,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 File change history is stored at: <https://github.com/mantidproject/mantid>
 */
-class DLLExport LoadInstrument : public API::Algorithm {
+class DLLExport LoadInstrument : public API::TriviallyParallelAlgorithm {
```

Comment:
Why is this change necessary for LoadInstrument?

Reply:
It is used by `LoadEventNexus` to load the instrument and must thus support MPI to have MPI support in `LoadEventNexus`. The same goes for the other modified algorithms.

Reply:
Yes. Since the instrument itself will not be distributed, we must ensure that it is fully loaded on all ranks that need it. I think the `TriviallyParallelAlgorithm` will work correctly in this case. No other changes needed.